cmd/di.go

```diff
@@ -43,6 +41,7 @@ import (
 	"github.com/mysteriumnetwork/node/services/openvpn"
 	openvpn_service "github.com/mysteriumnetwork/node/services/openvpn/service"
 	"github.com/mysteriumnetwork/node/session"
+	"path/filepath"
```

Comment:
This does not look goimports formatted.

Reply:
`bin/check_goimports: line 23: goimports: command not found`
hm, looks like we have broken `goimports` check.

Reply:
Created [issue](https://github.com/mysteriumnetwork/node/issues/450) for fixing goimports on the CI.

Reply:
Fixed code.